packages/appstore/src/components/modals/cashier-wallet-modal/cashier-wallet-modal-header.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Badge, GradientBackground, Icon, Text, Watermark } from '@deriv/components';
+import { formatMoney } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import { getWalletCurrencyIcon } from 'Constants/utils';
+import { getCashierWalletModalBackgrounds } from 'Constants/wallet-backgrounds';
+
+type TCashierWalletModalHeaderProps = {
+    balance?: string | number;
+    currency: string;
+    is_dark: boolean;
+    is_demo: boolean;
+    is_mobile: boolean;
+    closeModal: VoidFunction;
+    show_wallet_name: boolean;
+};
+
+const CashierWalletModalHeader = ({
+    balance,
+    closeModal,
+    currency,
+    is_dark,
+    is_demo,
+    is_mobile,
+    show_wallet_name,
+}: TCashierWalletModalHeaderProps) => {
+    const header_class_name = 'cashier-wallet-modal__header';
+
+    const getCloseIcon = React.useCallback(() => {
+        if (is_demo && is_dark) return 'IcAppstoreCloseLight';
+        if (is_demo && !is_dark) return 'IcAppstoreCloseDark';
+        if (is_dark) return 'IcAppstoreCloseDark';
+        return 'IcAppstoreCloseLight';
+    }, [is_dark, is_demo]);
+
+    const getStylesByClassName = React.useCallback(
+        (class_name: string) => {
+            return classNames(class_name, {
+                [`${class_name}-demo`]: is_demo,
+            });
+        },
+        [is_demo]
+    );
+
+    const getCurrencyIconSize = React.useCallback(() => {
+        // TODO: add p2p and payment_agent check
+        const is_square_icon =
+            ['btc', 'eth', 'ltc', 'usdt', 'eusdt', 'tusdt', 'ust', 'usdc', 'p2p', 'payment_agent'].includes(
+                currency.toLowerCase()
+            ) || is_demo;
+
+        const sizes = {
+            mobile: {
+                width: is_square_icon ? 64 : 48,
+                height: is_square_icon ? 40 : 48,
+            },
+            desktop: {
+                width: is_square_icon ? 128 : 64,
+                height: is_square_icon ? 80 : 64,
+            },
+        };
+
+        const size = is_mobile ? sizes.mobile : sizes.desktop;
+
+        return size;
+    }, [currency, is_demo, is_mobile]);
+
+    const getCurrencyIconProps = React.useCallback(() => {
+        const icon = getWalletCurrencyIcon(is_demo ? 'demo' : currency, is_dark);
+        const size = getCurrencyIconSize();
+
+        return { icon, ...size };
+    }, [currency, getCurrencyIconSize, is_dark, is_demo]);
+
+    const getBackgroundName = React.useCallback(() => {
+        if (is_demo) {
+            return 'demo';
+        }
+        return currency.toLowerCase();
+    }, [is_demo, currency]);
+
+    return (
+        <GradientBackground {...getCashierWalletModalBackgrounds(getBackgroundName())}>
+            <Watermark
+                image='url(/packages/appstore/src/public/images/wallet-demo-bg.svg)'
+                // image='url(https://i.postimg.cc/jqGY1q6R/Style-white.png)' - works fine
```

Comment:
@heorhi-deriv You need to import the file instead, You can use `require` or import the SVG file into a variable.
```ts
<Watermark image={`url(${require('../../../public/images/wallet-demo-bg.svg')})`} />
```
or
```ts
import image from '../../../public/images/wallet-demo-bg.svg';

<Watermark image={`url(${image})`} />
```